src/h5web/vis-packs/core/heatmap/HeatmapVis.tsx

```diff
@@ -101,11 +102,10 @@ function HeatmapVis(props: Props) {
             isIndexAxis: !ordinateValue,
             label: ordinateLabel,
           }}
-          imageRatio={layout === 'cover' ? cols / rows : undefined}
         >
           <TooltipMesh {...tooltipFormatters} guides="both" />
           <PanZoomMesh />
-          <Mesh abscissaDomain={abscissaDomain} ordinateDomain={ordinateDomain}>
+          <Mesh>
```

Comment:
Since the abscissa/ordinate domains are now left untouched by `AxisSystem`, we can retrieve them from `AxisSystemContext`.